Include/cpython/ceval.h

```diff
@@ -28,3 +28,5 @@ PyAPI_FUNC(Py_ssize_t) _PyEval_RequestCodeExtraIndex(freefunc);
 
 PyAPI_FUNC(int) _PyEval_SliceIndex(PyObject *, Py_ssize_t *);
 PyAPI_FUNC(int) _PyEval_SliceIndexNotNone(PyObject *, Py_ssize_t *);
+
+PyAPI_FUNC(void) _PyEval_DeactivateOpCache();
```

Comment:
Please move it to pycore_ceval.h (internal C API). Also, you should explicitly use ``(void)``, otherwise you can call it with any number of arguments (weird C API) if I recall correctly.

```suggestion
PyAPI_FUNC(void) _PyEval_DeactivateOpCache(void);
```